src/CareTogether.Api/OData/LiveODataModelController.cs

```diff
@@ -467,6 +484,28 @@ bool anonymize
 
             var people = familiesWithInfo.SelectMany(x => RenderPeople(x.Item1, x.Item2)).ToArray();
 
+            var locationUserAccess = (
+                await people
+                    .Select(async person =>
+                        (
+                            person,
+                            roles: await accountsResource.TryGetPersonRolesAsync(
+                                person.Family.Location.OrganizationId,
+                                person.Family.Location.Id,
+                                person.Id
+                            )
+                        )
+                    )
+                    .WhenAll()
+            )
+                .Where(person =>
+                    person.roles?.Contains(SystemConstants.ORGANIZATION_ADMINISTRATOR) ?? false
+                )
+                .Select(person => new LocationUserAccess(
+                    person.person.Id,
```

Comment:
Instead of person ID, obtain the user ID for the person (from `familiesWithInfo`).